build.gradle

```diff
@@ -34,7 +34,14 @@ dependencies {
     // or Jenkins plugin versions in Jenkins plugins that depend on this library.
     // Jenkins plugins that depend on this library should therefore version those dependencies compatibly with the versions below:
     // --rotte DEC 2019
-    implementation 'org.jenkins-ci.main:jenkins-core:2.359'
+    implementation('org.jenkins-ci.main:jenkins-core:2.359') {
+        // spring-security-web pulls in 'core' and 'crypto'.. we now replace that 5.7.2 with 5.7.5
+        // this still won't filter thru into the plugin however, which is unfortunate but this
+        // will at least also clean up annotation which depends on jenkins-common.
+        exclude group: 'org.springframework.security', module: 'spring-security-web'
+    }
```

Comment:
I would have made the same guess here (that line 41 wouldn't be required), but, again, you may have already proven otherwise.

Reply:
I've seen this approach used in more than one place, and it appeared to be the "most endorsed"... at least from my search results regarding the topic.

Reply:
moving to 2.377 removes the need to do this one....

Reply:
move to jekins 2.377 removes the need to play this game.